docs/source/en/package_reference/serialization.md

```diff
@@ -6,6 +6,116 @@ rendered properly in your Markdown viewer.
 
 `huggingface_hub` contains helpers to help ML libraries serialize models weights in a standardized way. This part of the lib is still under development and will be improved in future releases. The goal is to harmonize how weights are serialized on the Hub, both to remove code duplication across libraries and to foster conventions on the Hub.
 
+## DDUF file format
+
+DDUF is a file format designed for diffusion models. It allows saving all the information to run a model in a single file. This work is inspired by the GGUF format. `huggingface_hub` provides helpers to save and load DDUF files, ensuring the file format is respected.
```

Comment:
Do we wanna also talk a bit about the validation approach we have taken and that if need be we will update it?